lab/lab2/js/part3-application.js

```diff
@@ -31,25 +31,50 @@ var resetMap = function() {
   /* =====================
     Fill out this function definition
   ===================== */
+  map.removeLayer(myMarkers);
+  markerfilterd = [];
 };
 
 /* =====================
   Define a getAndParseData function to grab our dataset through a jQuery.ajax call ($.ajax). It
   will be called as soon as the application starts. Be sure to parse your data once you've pulled
   it down!
 ===================== */
+
+
 var getAndParseData = function() {
   /* =====================
     Fill out this function definition
   ===================== */
+  dat = $.ajax("https://raw.githubusercontent.com/CPLN692-MUSA611/datasets/master/json/philadelphia-crime-snippet.json").done(function(dat) {
+    myData = JSON.parse(dat);
+    console.log(myData);
+  });
 };
 
 /* =====================
   Call our plotData function. It should plot all the markers that meet our criteria (whatever that
   criteria happens to be — that's entirely up to you)
 ===================== */
+
+var markerfilter = function(arr) {
+  var markerfilterd = [], i = -1;
+  for (i = 0; i < arr.length; i++) {
+    //filter a certain type of crime within certain districts, and within certain Police Service Areas (PSA 1: checked, PSA 2: unchecked)
```

Comment:
I am unable to plot any points.
What inputs are you using to get markers?

![image](https://user-images.githubusercontent.com/8103418/52922104-41419b00-32ec-11e9-970b-3bd4d7b3ed72.png)

Reply:
The first and second numeric inputs are the range of districts you want to investigate; the string input is the type of crime; the boolean input indicates the PSA (Police Service Areas), checked means PSA 1, unchecked means PSA 2.
My inputs and plot are like this:
![image](https://user-images.githubusercontent.com/46826105/52922438-4bb16400-32ef-11e9-918f-935fa5788dfe.png)

Reply:
Ah, okay, I did not enter the crime name exactly as it expected. Okay. Nice work, thanks for clarifying!